frontend/packages/dev-console/src/components/resource-quota/ResourceQuotaAlert.tsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import { Label } from '@patternfly/react-core';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import {
+  useK8sWatchResource,
+  YellowExclamationTriangleIcon,
+} from '@console/dynamic-plugin-sdk/src/api/core-api';
+import { AppliedClusterResourceQuotaModel, ResourceQuotaModel } from '@console/internal/models';
+import { AppliedClusterResourceQuotaKind, ResourceQuotaKind } from '@console/internal/module/k8s';
+import { checkQuotaLimit } from '@console/topology/src/components/utils/checkResourceQuota';
+
+export interface ResourceQuotaAlertProps {
+  namespace: string;
+}
+
+export const ResourceQuotaAlert: React.FC<ResourceQuotaAlertProps> = ({ namespace }) => {
+  const { t } = useTranslation();
+  const [warningMessageFlag, setWarningMessageFlag] = React.useState<boolean>();
+  const [resourceQuotaName, setResourceQuotaName] = React.useState(null);
+  const [resourceQuotaKind, setResourceQuotaKind] = React.useState(null);
+
+  const [quotas, rqLoaded] = useK8sWatchResource<ResourceQuotaKind[]>({
+    groupVersionKind: {
+      kind: ResourceQuotaModel.kind,
+      version: ResourceQuotaModel.apiVersion,
+    },
+    namespace,
+    isList: true,
+  });
+
+  const [clusterQuotas, acrqLoaded] = useK8sWatchResource<AppliedClusterResourceQuotaKind[]>({
+    groupVersionKind: {
+      kind: AppliedClusterResourceQuotaModel.kind,
+      version: AppliedClusterResourceQuotaModel.apiVersion,
+      group: AppliedClusterResourceQuotaModel.apiGroup,
+    },
+    namespace,
+    isList: true,
+  });
+
+  const [totalRQatQuota, quotaName, quotaKind] = checkQuotaLimit(quotas);
+  const [totalACRQatQuota, clusterRQName, clusterRQKind] = checkQuotaLimit(clusterQuotas);
```

Comment:
these will be invoked even if rq for acrq is not loaded or if it's loaded but there is loadError. WDYT?

Reply:
Updated